baselines/ippo/config/ippo_ff_puffer.yaml

```diff
@@ -45,10 +45,12 @@ train:
   vf_coef: 0.5
   max_grad_norm: 0.5
   target_kl: null
-  checkpoint_interval: 5000
+  checkpoint_interval: 500 # Save policy every k iterations
```

Comment:
why change this?

Reply:
Figured it was a better default as 5000 only stored the policy before the run ends